features/promotion/managing_catalog_promotions/creating_catalog_promotion.feature

```diff
@@ -19,20 +19,41 @@ Feature: Creating a catalog promotion
         And this catalog promotion should be usable
 
     @api @ui @javascript
-    Scenario: Creating a catalog promotion
+    Scenario: Creating a enabled catalog promotion
         When I want to create a new catalog promotion
         And I specify its code as "winter_sale"
         And I name it "Winter sale"
         And I specify its label as "Winter -50%" in "English (United States)"
         And I describe it as "This promotion gives a 50% discount on all products" in "English (United States)"
         And I add rule that applies on variants "PHP T-Shirt" variant and "Kotlin T-Shirt" variant
         And I add action that gives "50%" percentage discount
+        And I make it available in channel "United States"
+        And I enable this catalog promotion
+        And I add it
+        Then there should be 1 new catalog promotion on the list
+        And it should have "winter_sale" code and "Winter sale" name
+        And "Winter sale" catalog promotion should apply to "PHP T-Shirt" variant and "Kotlin T-Shirt" variant
+        And it should have "50%" discount
+        And this catalog promotion should be usable
+        And "PHP T-Shirt" variant and "Kotlin T-Shirt" variant should be discounted
+
+    @api @ui @javascript
+    Scenario: Creating a disabled catalog promotion
+        When I want to create a new catalog promotion
+        And I specify its code as "winter_sale"
+        And I name it "Winter sale"
+        And I specify its label as "Winter -50%" in "English (United States)"
+        And I describe it as "This promotion gives a 50% discount on all products" in "English (United States)"
+        And I add rule that applies on variants "PHP T-Shirt" variant and "Kotlin T-Shirt" variant
+        And I add action that gives "50%" percentage discount
+        And I disable this catalog promotion
         And I add it
         Then there should be 1 new catalog promotion on the list
         And it should have "winter_sale" code and "Winter sale" name
         And "Winter sale" catalog promotion should apply to "PHP T-Shirt" variant and "Kotlin T-Shirt" variant
         And it should have "50%" discount
         And this catalog promotion should be usable
```

Comment:
I guess that disabled catalog promotion shouldn't be usable, it shouldn't dispatch an event 🤔